### ISSUE TITLE
Transparency: Gigablue LCD

### ISSUE THREAD

**Issue by AbuBaniaz** [picons/picons#71]
Just a quick question, have we changed anything with regards to the transparency of the picons?

Newer sets seem to show a blue background on the front panel of GigaBlue receivers. They are fine on the OSD. Other receivers are fine.

Apparently, older sets of picons were fine, but I'm sure these were manually renamed and not as the name suggests.

Thanks in advance 

Thread on WOS, I have also attached an older set there. Its bigger than 10MB so cannot attach here

https://www.world-of-satellite.com/showthread.php?60212-Gigablue-LCD-panel-Picon-background-not-transparent

**Comment by mrhotio** [comment]
Not that I know of...

**Comment by mrhotio** [comment]
According to that thread they found the issue to be related to 8bit transparency and a big in gigablue, I've always build them that way, because years ago that was the only way to get proper transparency and they are a lot smaller in filesize.

**Comment by mrhotio** [comment]
As can be seen here (https://github.com/picons/picons/blob/6e3927adf99f86571814491fe3676fbeaf1f12b5/2-build-picons.sh#L55) we put the picons thru pngquant to optimize the size, so I'm assuming that the builds that were hosted by OpenVix weren't build by us?

**Comment by AbuBaniaz** [comment]
They were built by us. But I think names might be wrong so leading to confusion that they worked  before and somehow changed. Not 100% sure though.

**Comment by mrhotio** [comment]
All I can tell about transparency is:

1) When I bought an VU+ DUO (this one https://s.s-bol.com/imgbase0/imagebase3/large/FC/0/9/6/2/9200000061072690.jpg), I had to use pngquant to get any transparency, most other boxes at the time suffered from the same limitation. There was no support for 32bit alpha transparency
2) Starting with some models or perhaps an update from OpenPli made support available for 32bit (IIRC, not sure what was the trigger), but 8bit transparency was still working fine and there was no reason to change that.
3) For the most part IIRC I've always released builds with 8bit transparency, because the outcry was large when for a reason I failed to install pngquant and they were released with 32bit.
4) So not sure what to do next, 8bit works for most stuff, releases are quite smaller in size,....just 1 model doesn't support it on their LCD screen....
